sdk/storage/azure-storage-blob/perf.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  - name: packageVersions
+    displayName: PackageVersions (regex of package versions to run)
+    type: string
+    default: '12|source'
```

Comment:
What does 12 mean here?

Reply:
This expression is a regex selecting which package versions to run.  Currently for Java, "source" means use whatever is in the pom.xml files in main (or the branch you are running the pipeline against).  We might change this for Java, to force all deps to use sources instead of pacakges.

"12" matches any package version that contains "12", so our GA storage packages.  Currently, the list of packages to use is listed in the azure-sdk-tools repo, and manually updated by me after releases:

https://github.com/Azure/azure-sdk-tools/blob/main/tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/tests.yml#L66

So "12" really means "latest GA packages".  I am looking at updating this logic, so you can specify a string like "latest" and the perf automation will dynamically determine the latest GA packages by querying the package managers.